app/controllers/api/message2s_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::Message2sController < ApplicationController
+  def random_greeting
+    message = Message.order('RANDOM()').first
+    render json: { greeting: message.content }
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229120316-539da540-dba7-49c6-bf36-59619f74045a.png)

- [ ] It would be nice if you fix this error I am getting from the backend :+1: because of this error your application is not rendering a JSON object :thinking: